setup.py

```diff
@@ -38,4 +38,7 @@
     setup_requires=['setuptools_scm'],
     namespace_packages=['mlx'],
     keywords=['robot', 'robotframework', 'sphinx', 'traceability'],
+    entry_points={
+        'console_scripts': ['mlx.robot2rst = mlx.robot2rst:main']
```

Comment:
Add `robot2rst` as well since we want to make it standard

Reply:
Done in ad8d8c7d6e16b94aa11e791dca7d8b9d5a5d0860

Reply:
added ad8d8c7d6e16b94aa11e791dca7d8b9d5a5d0860